src/Atlas.Client/Pages/Admin/Members/Edit.razor

```diff
@@ -37,47 +37,47 @@ else
                 <div class="col-md-6">
                     <div class="form-group">
                         <label class="control-label font-weight-bold">@Loc["Status"]:</label>
-                        <label class="">@Model.Info.Status</label>
```

Comment:
Why is this class causing issues? 
And anyway you left the property empty.
Please remove this changes and I'll merge the pull request.

Reply:
having `class=""`  cause a compile error for me

Reply:
for example this:

`<label class="">@Model.Info.Status</label>`

cause this:

![image](https://user-images.githubusercontent.com/40014477/91698764-d2fabb80-eba5-11ea-8b5f-717d77cf8504.png)